### DISCUSSION TITLE
Introductions 👋

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch9.buidlguidl.com#1] [Announcements]
👋 Welcome Batch 9 Builders! 

It's Eda here, I've been part of the BuidlGuidl for almost 2 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in web3. I spent most of my time working on developer education and creating content/documentation.

Hope that the batch will be a very fun project for you all to go through. If you have any questions, feel free to ask anytime!

**Comment by derrekcoleman** [comment]
Hi everyone! I'm Derrek, "Batch Dad" of the group 🧡 I've known the BG crew for a few years now and have been supporting these batches this year. I used to be a high school physics teacher before working at ETHDenver and now [BOB](https://www.gobob.xyz/).

I look forward to getting to know you all better this month! Like Eda said, let us know if you have any questions 🤝

**Comment by fedeloterstein** [comment]
Hi, I’m [Fedev](https://x.com/fedev_), a full stack developer specializing in front. I’m from Argentina and have been exploring web3 for the past 3 years. I love traveling and hacking at EthGlobal events whenever I can. Looking forward to collaborating on this repo and building something amazing together!

**Reply by derrekcoleman** [reply]
Welcome, Fedev! I can't wait to see what you buidl 💪

**Comment by phipsae** [comment]
Hi everyone,

My name is Philip, and I've been a proud member of BuidlGuidl for the past year now. I suport the batches program. I'm a developer with a passion for learning new concepts.

I'm excited to have you here. If you have any questions, feel free to reach out to us!

Enjoy your time here! 😊

**Comment by mapfumo** [comment]
Hello Everyone,
This is Tony, making a transition from network engineering. I have been exploring blockchain technologies for the last year. There's so much to learn and I'm glad I found BG. I'm looking forward to contributing and learning.

**Reply by phipsae** [reply]
Welcome @mapfumo! We are glad to have you and we are looking forward to see what you are gonna build!

**Reply by derrekcoleman** [reply]
It's great to have you, Tony! We're lucky to have someone with your experience joining web3 and BG 🤝

**Comment by vzaporozky** [comment]
Hi everyone,
My name is Vlad, and I have experience with Front-end. I have been studying blockchain for last 2 months. I'm glad to participate in this project and cooperate with you!    (>‿◠)✌

**Reply by derrekcoleman** [reply]
Welcome to BG, Vlad! Only two months?! I'm so glad you found us so early ۹(ÒہÓ)۶

**Comment by arjanjohan** [comment]
gm gm,
I'm Arjan and I discovered BuidlGuidl via the Scaffold-ETH 2 project, which is now the basis for almost al my hackathon projects. I have a background in economics and back-end development, and now I'm learning smart contract and front-end development.

**Reply by derrekcoleman** [reply]
What a powerful combo economics and app development are! As Charlie Munger said, "Never, ever, think about something else when you should be thinking about the power of incentives"

**Comment by ishtails** [comment]
heyy it's kartik... I'm a full stack web developer trying out web3 for the first time :))

one of my friends suggested i should start here and I'm really excited to start building something cool with y'all 🙌

**Reply by derrekcoleman** [reply]
It's good to have you, kartik! Welcome to web3 dev - you found a great starting place 🏠

**Comment by danitome24** [comment]
Hi all,

My name is Daniel and I'm from Spain. I'm a backend developer and for the past 7 years I've been working with Php as main language. discovered web3 development through experimenting with crypto. Recently, I completed some tutorials: the first one on Udemy, then Cyfin Updraft, where I heard about SpeedrunEthereum. That's how I found BG.

Looking forward to improving with all of you! :heart:

**Reply by derrekcoleman** [reply]
Welcome, Daniel! I'm a huge fan of Patrick and the Cyfrin team. Many of us entered web3 through Patrick, Austin, and BuidlGuidl - you're among friends.

**Comment by d4rm5** [comment]
Hi everyone! My name is Dante, I'm from Argentina and I study systems engineering. I know crypto since 2017 and I love Ethereum since 2021. This year I want to practice a lot about smart contract development, so I'm here.

**Reply by phipsae** [reply]
Welcome Dante! It’s a great place to start. Nice to have you here :)

**Reply by derrekcoleman** [reply]
Welcome to the batch, Dante! You found the right community to practice smart contract development with 💪

**Comment by 0xabdulkadir** [comment]
Hello, my name is Abdulkadir. I am from Turkey, and I am studying to become a doctor. First, I learned Python and JavaScript, then I learned Solidity. I watch Austin Griffith’s playlists, which help me a lot. Actually I opened Github for you :)
Thanks for the opportunity

**Reply by phipsae** [reply]
Welcome Abdulkadir! Nice to have you on board!
